### DISCUSSION TITLE
How can I minimize egui window to the system tray?

### DISCUSSION THREAD

**Discussion by YassinEldeeb** [emilk/egui#1388] [Q&A] [answered]
I wonder if this functionality is available in Egui cause I saw it in [tauri](https://tauri.studio/docs/guides/system-tray).

![Screenshot 2022-03-20 144416](https://user-images.githubusercontent.com/72823042/159162912-e4d24a41-6566-4b6b-8b32-a85b2b64939f.png)

**Comment by emilk** [comment, marked as answer]
It is not exposed in `eframe` yet. If you can find it in `winit` it can be added pretty easily. If not, perhaps it can be found in a 3rd party crate (or in https://github.com/tauri-apps/tao)

**Reply by YassinEldeeb** [reply]
That's helpful, thank you.

**Reply by cr1901** [reply]
This is also functionality I would like, and it appears the [`trayicon`](https://github.com/Ciantic/trayicon-rs) crate works fine w/ `winit`. At least, I compiled the example app locally, and I got a system tray like I wanted. I understand that exposing this in `eframe` would probably be ideal for lots of people, but seeing that a lot of ppl lack bandwidth lately (including myself :P), what would be the minimum amount of work to get `trayicon` to work with `egui` if I use the `winit` backend?

(Also related... does an example of using `egui` with `winit` exist?)

**Reply by emilk** [reply]
`eframe` uses `egui-winit`, so all example in https://github.com/emilk/egui/tree/master/examples uses winit by proxy. There is also a low-level example at https://github.com/emilk/egui/blob/master/egui_glow/examples/pure_glow.rs

**Comment by Gremious** [comment]
This is the first question that shows up when you google "egui minimize to tray" so have a real answer here: https://github.com/emilk/egui/discussions/737#discussioncomment-8830140
